conda_forge_tick/migrators/version.py

```diff
@@ -197,21 +222,27 @@ def migrate(
         **kwargs: Any,
     ) -> "MigrationUidTypedDict":
         version = attrs["new_version"]
-
-        with open(os.path.join(recipe_dir, "meta.yaml")) as fp:
-            raw_meta_yaml = fp.read()
-
-        updated_meta_yaml, errors = update_version(
-            raw_meta_yaml,
-            version,
-            hash_type=hash_type,
-        )
+        recipe_dir = Path(recipe_dir)
+        recipe_file = recipe_dir / "meta.yaml"
+        recipe_yaml = recipe_dir / "recipe.yaml"
```

Comment:
This naming is very confusing. Both are recipes, both are files, both are in yaml. I suggest either `meta_yaml` / `recipe_yaml` or something like `recipe_file_v0` / `recipe_file_v1`

Reply:
If you need to assign something to the same variable, then use `recipe_file` for that variable, and where you need to distinguish, use `recipe_file_v0` and `recipe_file_v1`. The current variable naming is not suitable

Reply:
It's weird - your comment wasn't there when I wrote mine, and I got the notification afterwards too. Presumably you had written the comment earlier but only submitted the review later? Nothing important, was just wondering why it looks like I ignored your message 😅

Reply:
However, the naming here is still bad. Why should v0 get `file` and v1 get a completely different name? Please use consistent naming, like `recipe_file`, `recipe_file_v0` and `recipe_file_v1`.

Something like
```suggestion
        recipe_file = None  # will be determined based on version in use
        recipe_file_v0 = recipe_dir / "meta.yaml"
        recipe_file_v1 = recipe_dir / "recipe.yaml"
        if recipe_file_v0 .exists():
            recipe_file = recipe_file_v0
            [...]
       [...]
       recipe_file.write_text(updated_meta_yaml)
```